### DISCUSSION TITLE
Connect for Chrome

### DISCUSSION THREAD

**Discussion by testacc992** [3kh0/ext-remover#481] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know how to bypass "Connect for Chrome" blocker or linewize? 

![image](https://user-images.githubusercontent.com/94496154/216436778-bcd588cb-b74e-4427-89cd-63cf9c79c6d6.png)
![image](https://user-images.githubusercontent.com/94496154/216436943-b4d26bdd-4e21-4624-b0dd-7141e93f7519.png)

**Comment by 3kh0** [comment, marked as answer]
From what I know, here are your options.
- Downgrade and LTBEEF
- unenroll with shimmer
- thats it
